### DISCUSSION TITLE
Is it normal that the "latest" tag refers to an alpha release?

### DISCUSSION THREAD

**Discussion by akkadaya** [react-native-community/cli#2539] [Q&A] [answered]
I was creating a new project and it asked me to use an alpha version. checked npmjs and saw this

<img width="500" alt="image" src="https://github.com/user-attachments/assets/b73f2a56-5c02-4ebb-8206-1aacbf13716a">

Is it normal that the "latest" tag refers to an alpha release?

**Comment by thymikee** [comment, marked as answer]
Fixed that, thanks for noticing!
